ansible/roles/jenkins-worker/templates/start.j2

```diff
@@ -7,6 +7,6 @@ export OSTYPE=osx
 export ARCH=x64
 export DESTCPU=x64
 
-PATH="/usr/local/opt/ccache/libexec:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin" {{ java_path[os] }} -Xmx{{ server_ram|default('128m') }} \
+PATH="/usr/local/opt/ccache/libexec:/usr/local/opt/python3/Frameworks/Python.framework/Versions/Current/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin" {{ java_path[os] }} -Xmx{{ server_ram|default('128m') }} \
```

Comment:
This change is because, for some reason I am unable to determine, `test-orka-macos10.14-x64-1` ends up putting `tap2junit` in `/usr/local/opt/python3/Frameworks/Python.framework/Versions/Current/bin`. With the same ansible scripts on the other two macOS 10.14 hosts, `tap2junit` gets written to `/usr/local/bin` as expected 🤷.